.circleci/config.yml

```diff
@@ -125,6 +139,132 @@ jobs:
+
+  test-assembly-apt:
+    machine: true
+    working_directory: ~/grakn
+    steps:
+      - checkout
+      - install-bazel
+      - run:
+          command: |
+            echo $(date +%s)-$(cat VERSION)-$CIRCLE_SHA1 > VERSION
+            echo VERSION=$(cat VERSION)
+      - run: bazel build //bin:assemble-apt
+      - run: bazel build //server:assemble-apt
+      - run: bazel build //console:assemble-apt
+      - run: sudo dpkg -i bazel-bin/bin/grakn-core-bin__all.deb
+      - run: sudo dpkg -i bazel-bin/server/grakn-core-server__all.deb
+      - run: sudo dpkg -i bazel-bin/console/grakn-core-console__all.deb
+      - run: sudo chown -R circleci:circleci /opt/grakn/ # TODO: how do we avoid having to chown?
+      - run: nohup grakn server start
+      - run: bazel test //test-deployment:test-deployment --test_output=streamed --spawn_strategy=standalone --cache_test_results=no
+      - run: grakn server stop
+      # TODO: how to get repo.grakn.ai apt repository to not ignore commit id in the version? that way we don't have to add date +%s
+      - run: bazel run //bin:deploy-apt -- test $REPO_GRAKN_USERNAME $REPO_GRAKN_PASSWORD
+      - run: bazel run //console:deploy-apt -- test $REPO_GRAKN_USERNAME $REPO_GRAKN_PASSWORD
+      - run: bazel run //server:deploy-apt -- test $REPO_GRAKN_USERNAME $REPO_GRAKN_PASSWORD
+      - persist_to_workspace:
+          root: ~/grakn
+          paths:
+            - VERSION
+
+  test-deployment-apt:
+    machine: true
+    working_directory: ~/grakn
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/circleci-workspace
+      - install-bazel
+      - run: echo "deb [ arch=all ] https://repo.grakn.ai/repository/test-deb/ trusty main" | sudo tee -a /etc/apt/sources.list.d/grakn-core.list
+      - run: sudo apt-key adv --keyserver keyserver.ubuntu.com --recv 8F3DA4B5E9AEF44C
+      - run: sudo apt update
+      - run:
+          command: |
+            mv ~/circleci-workspace/VERSION VERSION
+            cat VERSION
+      - run:
+          command: |
+            sudo apt install grakn-core-server=$(cat VERSION) grakn-core-console=$(cat VERSION) # TODO: install the right version
+            sudo chown -R circleci:circleci /opt/grakn/ # TODO: how do we avoid having to chown?
+      - run: nohup grakn server start
+      - run: bazel test //test-deployment:test-deployment --test_output=streamed --spawn_strategy=standalone --cache_test_results=no
+      - run: grakn server stop
+
+  test-assembly-rpm:
+    machine: true
+    working_directory: ~/grakn
+    steps:
+      - checkout
+      - install-bazel
+      - run: sudo apt install rpm
+      - run:
+          command: |
+            echo $(cat VERSION)-$CIRCLE_SHA1 > VERSION
+            sed -i -e 's/-/_/g' VERSION
+            cat VERSION
+      - run: bazel build //bin:assemble-rpm
+      - run: bazel build //server:assemble-rpm
+      - run: bazel build //console:assemble-rpm
```

Comment:
I think we don't need to call `bazel build <target>`, given that `bazel run <target>` will trigger them too. It will simplify the CI file. Unless if you need to do something in the middle, like in `test-assembly-apt` above.

Reply:
good point, fixed in cc8d8f2a3b2b4bf079e842c37e9340a2df58459e

Reply:
good point, fixed in cc8d8f2a3b2b4bf079e842c37e9340a2df58459e